rules.adoc

```diff
@@ -578,6 +573,12 @@ magnetic interference. However, the robots need to be constructed in a way
 which allows them to work in conditions that are not perfect (i.e. by not
 relying on compass sensors or specific lightning conditions).
 
+IMPORTANT: At RCJ Eindhoven 2013 at the same location that 2024 will be magnetic fields
```

Comment:
@dbscoach I generally don't really like including venue-specific notes like this but in this case it can be viewed as an extension of the larger point of "do not rely on consistent magnetic fields".